src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -481,15 +487,25 @@ private PaymentInfo buildPaymentInfo(final Account account, final UUID kbPayment
         final String ccType = PluginProperties.getValue(PROPERTY_CC_TYPE, paymentMethodsRecordCcType, properties);
         final String recurringDetailId = PluginProperties.findPluginPropertyValue(PROPERTY_RECURRING_DETAIL_ID, mergedProperties);
 
+        final PaymentInfo paymentInfo;
         if (recurringDetailId != null) {
-            return buildRecurring(paymentMethodsRecord, paymentProvider, mergedProperties);
+            paymentInfo = buildRecurring(paymentMethodsRecord, paymentProvider, mergedProperties);
         } else if ("sepadirectdebit".equals(ccType)) {
-            return buildSepaDirectDebit(nonNullPaymentMethodsRecord, paymentProvider, mergedProperties);
+            paymentInfo = buildSepaDirectDebit(nonNullPaymentMethodsRecord, paymentProvider, mergedProperties);
         } else if ("elv".equals(ccType)) {
-            return buildElv(nonNullPaymentMethodsRecord, paymentProvider, mergedProperties);
+            paymentInfo = buildElv(nonNullPaymentMethodsRecord, paymentProvider, mergedProperties);
         } else {
-            return buildCreditCard(nonNullPaymentMethodsRecord, paymentProvider, mergedProperties);
+            paymentInfo = buildCreditCard(nonNullPaymentMethodsRecord, paymentProvider, mergedProperties);
         }
+
+        paymentInfo.setContinuousAuthenticationEnabled(isContinuousAuthenticationEnabled(mergedProperties));
+
+        return paymentInfo;
+    }
+
+    private boolean isContinuousAuthenticationEnabled(final Iterable<PluginProperty> mergedProperties) {
+        final String contAuth = PluginProperties.findPluginPropertyValue(PROPERTY_CONTINUOUS_AUTHENTICATION, mergedProperties);
+        return Boolean.parseBoolean(contAuth);
```

Comment:
In my eyes not enforcing the cvv for none recurring payments is dangerous and an open door for fraud (but maybe that's only typical german angst regarding to creditcard payments ;)).

Most purchases (~60%) in our old system are without recurrings. So that's sill the default.

At least in the Adyen documentation contAuth is also always used in conjunction with recurrings. 
In the end this is a business decision (is a potential higher volume of checkouts enough to compensate for all the additional fraud and legal problems) someone should explicitly take (with default = true we would do that for him).